--- ./scripts/pyi_rth_pkgutil.py	2021-07-15 18:29:28.109170197 -0400
+++ ./venv/lib/python3.9/site-packages/PyInstaller/hooks/rthooks/pyi_rth_pkgutil.py	2021-07-15 18:30:08.943797191 -0400
@@ -68,7 +68,7 @@
         # Only single path is supported, and it must start with
         # sys._MEIPASS
         pkg_path = os.path.normpath(path[0])
-        assert pkg_path.startswith(SYS_PREFIX)
+        assert pkg_path.startswith(str(SYS_PREFIX))
         # Construct package prefix from path...
         pkg_prefix = pkg_path[SYS_PREFIXLEN:]
         pkg_prefix = pkg_prefix.replace(os.path.sep, '.')
